org.cloudcompare.CloudCompare.yaml

```diff
@@ -369,109 +324,120 @@ modules:
       - -DBUILD_tracking=OFF
       - -DBUILD_visualization=OFF
 
-  - name: yasm
+  - name: LASzip
     sources:
       - type: archive
-        url: https://github.com/yasm/yasm/releases/download/v1.3.0/yasm-1.3.0.tar.gz
-        sha256: 3dce6601b495f5b3d45b59f7d2492a340ee7e84b5beca17e48f862502bd5603f
+        url: https://github.com/LASzip/LASzip/releases/download/3.4.3/laszip-src-3.4.3.tar.gz
+        sha256: 53f546a7f06fc969b38d1d71cceb1862b4fc2c4a0965191a0eee81a57c7b373d
         x-checker-data:
           type: anitya
-          project-id: 5286
+          project-id: 241499
           stable-only: true
-          url-template: https://github.com/yasm/yasm/releases/download/v$version/yasm-$version.tar.gz
-    buildsystem: autotools
+          url-template: https://github.com/LASzip/LASzip/releases/download/$version/laszip-src-$version.tar.gz
+    buildsystem: cmake-ninja
     builddir: true
     config-opts:
-      - --disable-shared
-
-  - name: MPIR
-    sources:
+      - -DCMAKE_BUILD_TYPE=Release
+     
+  - name: OpenCV	
+    sources: 
```

Comment:
There are trailing whitespace characters here.

Reply:
thanks. I will fix that ASAP